### PR TITLE
Fixed blocker code issue reported by sonar

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-605

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java [hunk 1]
```diff
@@ -45,13 +45,13 @@ public VerificationResult verifyZip(String blobName, InputStream zipSource) {
...
-            return error(ErrorCode.ERR_SIG_VERIFY_FAILED, "Invalid signature");
+            return getError(ErrorCode.ERR_SIG_VERIFY_FAILED, "Invalid signature");
         } catch (InvalidZipArchiveException ex) {
             logger.info("Invalid zip archive. Blob name: {}", blobName, ex);
-            return error(ErrorCode.ERR_ZIP_PROCESSING_FAILED, "Invalid zip archive");
+            return getError(ErrorCode.ERR_ZIP_PROCESSING_FAILED, "Invalid zip archive");
         } catch (IOException ex) {
             logger.info("Error occurred when verifying file. Blob name: {}", blobName, ex);
-            return error(ErrorCode.ERR_ZIP_PROCESSING_FAILED, "Invalid zip archive");
+            return getError(ErrorCode.ERR_ZIP_PROCESSING_FAILED, "Invalid zip archive");
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java:54]: VerificationResult can be completely static. 3 static instance?

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java [hunk 2]
```diff
@@ -70,7 +75,7 @@ public static VerificationResult ok() {
             return new VerificationResult(true, null, null);
         }
 
-        public static VerificationResult error(ErrorCode error, String reason) {
+        public static VerificationResult getError(ErrorCode error, String reason) {
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java:78]: we can remove this?

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java:78]: or private

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java:76]: it is used in test classes

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java:76]: can use static instance?

**Review by mdayican — APPROVED** [review]

**Pull request merged**